src/Reflex/Dom/SemanticUI/Dropdown.hs

```diff
@@ -24,34 +25,23 @@ import qualified Data.Map as M
 import           Data.Monoid
 import           Data.Text (Text)
 import qualified Data.Text as T
-import           GHCJS.DOM.Types hiding (Event, Text)
-#ifdef ghcjs_HOST_OS
-import           GHCJS.Foreign.Callback
-import           GHCJS.Foreign
-import           GHCJS.Marshal.Internal
-import           GHCJS.Types
-#endif
 import           Reflex
 --import           Reflex.Host.Class
 import           Reflex.Dom hiding (fromJSString)
 ------------------------------------------------------------------------------
+import           GHCJS.Compat
 import           Reflex.Dom.SemanticUI.Common (tshow)
 ------------------------------------------------------------------------------
 
 
 ------------------------------------------------------------------------------
 activateSemUiDropdown :: Text -> IO ()
-#ifdef ghcjs_HOST_OS
 activateSemUiDropdown = js_activateSemUiDropdown . toJSString
 
-foreign import javascript unsafe
-  "$($1).dropdown({fullTextSearch: true});"
-  js_activateSemUiDropdown :: JSString -> IO ()
-#else
-activateSemUiDropdown =
-  error "activateSemUiDropdown: can only be used with GHCJS"
```

Comment:
This is where we could have added a JSaddle implementation to make the GHCS version work.